ctapipe/image/cleaning.py

```diff
@@ -222,7 +228,7 @@ def number_of_islands(geom, mask):
     return num_islands, island_labels
 
 
-def _apply_time_delta_cleaning(
+def apply_time_delta_cleaning(
     geom, mask, arrival_times, min_number_neighbors, time_limit
 ):
     """ Remove all pixels from selection that have less than N
```

Comment:
should perhaps change the docstring to "identify all pixels", since it doesn't remove anything.